Signal/src/util/DeviceSleepManager.swift

```diff
@@ -71,7 +71,7 @@ import Foundation
 
         if UIApplication.shared.isIdleTimerDisabled != shouldBlock {
             if shouldBlock {
-                Logger.info("\(self.TAG) \(#function): Blocking sleep")
```

Comment:
If there's ever an issue, I imagine it could be helpful to know *what* is blocking the device from going to sleep.